sjb/config/test_cases/azure_build_node_image_centos.yml

```diff
@@ -15,6 +15,8 @@ actions:
     title: "determine the resource group"
     script: |-
       echo "RESOURCE_GROUP=ci-$JOB_NAME-$BUILD_NUMBER" | ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel "cat >>/etc/environment"
+      ORIGIN_VERSION=$(repoquery --disablerepo \*  --repofrompath origin,$(curl -qs https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/master/.latest-rpms) --enablerepo origin -q origin --qf '%{VERSION}' 2>/dev/null)
```

Comment:
I'd be happy to hard-code ORIGIN_VERSION to 'master' in this case to avoid calculating it.  @kargakis ok with you?

Reply:
I meant to imply that in azure_build_node_image_centos_39.yml we would hard-code it in the script to '3.9'.
ach, I guess I don't mind either way either.

Reply:
Yeah, release branch jobs should hardcode to the actual version.